src/hero/hero.macros.html

```diff
@@ -12,14 +12,14 @@
   * @param {string} [imgix_class=imgix-fluid--nocss] - Class name for custom imgix.js fluid targeting/manipulations
   * @param {string} [class] - Class name for style or JS targeting
   * @param {string} [id] - ID for JS targeting
-  *
-  * todo: If no content, use a caller() block
 #}
 {% macro hero(heading, content=null, background_path=null, featured=false, imgix_class='imgix-fluid--nocss', class=null, id=null) %}
 <header class="hero{% if featured %} hero--featured{% endif %} {{ imgix_class }}{% if class %} {{ class }}{% endif %}"{% if id %} id="{{ id }}"{% endif %}{% if background_path %} data-src="https://casper.imgix.net/{{ background_path }}"{% endif %}>
   <h1 class="hero-heading">{{ heading }}</h1>
   {% if content %}
   <h2 class="hero-subheading">{{ content }}</h2>
+  {% else %}
+  {{ caller() }}
   {% endif %}
 </header>
 {% endmacro %}
```

Comment:
Do you want new lines for control loops or between elements?  Or both?  I'm fine either way but if we're going to have more whitespace in the macro, I think @josephco was saying we have to minify the content (which we should probably do anyway).